.github/workflows/gatsby-build.yaml

```diff
@@ -0,0 +1,17 @@
+name: Build and Deploy
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Gatsby Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - run: yarn install
+      - run: yarn deploy
```

Comment:
Does this deploy to the `gh-pages` branch only?

Reply:
@badboy Yes the `yarn deploy` runs the `gh-pages` command which previously did that for us manually :D The documentation is still in the README although possibly no longer needed.